src/app/cases/viewer/case-viewer.component.ts

```diff
@@ -105,4 +106,8 @@ export class CaseViewerComponent implements OnInit, OnDestroy {
       return acc.concat(tab.fields);
     }, []);
   }
+
+  isDraft(): boolean {
```

Comment:
Any luck with moving this method into CaseDetails? I still believe that all logic to determine if a case details is a draft would be better placed in one place only

Reply:
We need this since used in the HTML page. Centralisation is addressed in the next [PR](https://github.com/hmcts/ccd-case-management-web/pull/183)